platform/broadcom/sai.mk

```diff
@@ -1,12 +1,16 @@
-BRCM_SAI = libsaibcm_6.0.0.10-1_amd64.deb
-$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/6.0/master/XGS/libsaibcm_6.0.0.10-1_amd64.deb?sv=2015-04-05&sr=b&sig=5fW6otWPCPrsSIXeomEcllS8sEDXMo0YOD8UhB370U0%3D&se=2035-07-22T20%3A50%3A51Z&sp=r"
-BRCM_SAI_DEV = libsaibcm-dev_6.0.0.10-1_amd64.deb
+LIBSAIBCM_VERSION = 6.0.0.10-1
+LIBSAIBCM_BRANCH_NAME = REL_$(shell echo $(LIBSAIBCM_VERSION) | cut -d. -f1,2)
```

Comment:
Currently BRCM does not have a standard way to name their releases.  We have been trying to follow their naming where we bring in their code to libsaibcm side.  Here are the names so far:
For very old releases such as 20181130:
       3.5-201811
For 20191130:
      /rita/3_7
For 20201231:
     REL_4.3.5.1
For 20210630:
     REL_5.2
For master:
     REL_6.0

So not sure what you proposed here may cause future release from BRCM naming issue?
REL_4.3.5
REL_

Reply:
@gechiang , can we simply use REL_6.0? If the branch change, we need to update it, otherwise, we only update the version.